scripts/install/jellyfin.sh

```diff
@@ -19,15 +19,6 @@ app_port_https="$(shuf -i 10001-32001 -n 1)" && while [[ "$(ss -ln | grep -co ''
 # Get our exertnal IP 4 address and set it as a variable.
 ip_address="$(curl -s4 icanhazip.com)"
```

Comment:
Is this necessary? I'd skip the echos at the end altogether

Reply:
How do i know which port to access the application on?

With nginx i can guess the url to be /jellyfin

Without nginx i can't guess the random port.

Reply:
Only thing I see it in are the echos at the bottom, which I think are not necessary, barely any other apps announce where they are accessible with an actual IP. It could also check for the domain, it could as well be inaccessible outside the local network.

Reply:
/resolved